### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by ViktoriiaMarkus**
- [DEMO LINK](https://ViktoriiaMarkus.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://ViktoriiaMarkus.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +7,66 @@
...
-    <h1>Moyo header</h1>
+  <!-- #endregion -->
+  <body id="home">
+
+    <header class="header">
+
+      <a href="#home" class="logo">
+        <img class="logo__img"
+        src="images/logo.png"
+        alt="logo">
```
Comment (xiimiik) [inline src/index.html:23]: ckecklist '13 ❗

File: src/index.html [hunk 2]
```diff
@@ -6,9 +7,66 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <!-- #endregion -->
+  <body id="home">
+
+    <header class="header">
+
```
Comment (xiimiik) [inline src/index.html:19]: empty lines should only be between children, not between parent and child

File: src/index.html [hunk 3]
```diff
@@ -6,9 +7,66 @@
...
+        <a href="#smartphones"
+         class="nav__link">
+          Smartphones
+        </a>
+
+        <a href="#laptops-and-computers"
+         data-qa="hover"
+         class="nav__link is-active">
+          Laptops & computers
+        </a>
```
Comment (xiimiik) [inline src/index.html:46]: ckecklist '13 ❗

File: src/index.html [hunk 4]
```diff
@@ -6,9 +7,66 @@
...
+      <a href="#home" class="logo">
+        <img class="logo__img"
+        src="images/logo.png"
+        alt="logo">
+      </a>
+
+      <nav class="nav">
+        <a href="#apple"
+         class="nav__link">
+          Apple
```
Comment (xiimiik) [inline src/index.html:29]: if there are less than 3 attributes, then there is no need to transfer

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,59 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  text-transform: uppercase;
+  background-color: #fff;
+  padding: 10px 50px;
```
Comment (xiimiik) [inline src/style.css:10]: the clickable area of the link should be the full height of the header, so 10px is not needed here

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,59 @@
...
+.logo__img {
+  height: 40px;
+  width: 40px;
+  display: block;
+}
+
+.nav {
+  display: inline-block;
+  background-color: #fff;
+  height: 100%;
```
Comment (xiimiik) [inline src/style.css:25]: not necessary

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,59 @@
...
+}
+
+.nav__link {
+  cursor: pointer;
```
Comment (xiimiik) [inline src/style.css:29]: not necessary, its default value for links

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,59 @@
...
+}
+
+.nav__link.is-active::after {
+  background-color: #00acdc;
+  transform: scaleX(1);
+}
+
+.nav__link.is-active {
+  color: #00acdc;
 }
```
Comment (xiimiik) [inline src/style.css:59]: the link with blue color and line below is an active link. It should have `class="is-active"` and relevant styles.
these styles should be for `is-active`, not for `.nav__link`

**Review by xiimiik — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 5 above)
Comment (ViktoriiaMarkus) [reply src/style.css:10]: Hello, it`s not like clickable area, it is like space between head of page and our header and the same 50px like space between left and right border to our header

**Comment by ViktoriiaMarkus** [comment]
Thank you that check my mistakes, I tried to fix them. I pin links below one more time 
- [DEMO LINK](https://ViktoriiaMarkus.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://ViktoriiaMarkus.github.io/layout_moyo-header/report/html_report/)

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,50 @@
 body {
+  font-family: "Roboto", sans-serif;
```
Comment (denys-danyliuk) [inline src/style.css:2]: 3'rd point in checklist

File: src/index.html [hunk 10]
```diff
@@ -6,9 +7,44 @@
...
+         Laptops & computers
+        </a>
+
+        <a href="#gadgets" class="nav__link">Gadgets</a>
+
+        <a href="#tablets" class="nav__link">Tablets</a>
+
+        <a href="#photo" class="nav__link">Photo</a>
+
+        <a href="#video" class="nav__link">Video</a>
```
Comment (denys-danyliuk) [inline src/index.html:45]: use ul component for this list

File: src/index.html [hunk 11]
```diff
@@ -6,9 +7,44 @@
...
+
+        <a href="#samsung" class="nav__link">Samsung</a>
+
+        <a href="#smartphones" class="nav__link">Smartphones</a>
+
+        <a href="#laptops-and-computers"
+         data-qa="hover"
+         class="nav__link is-active">
+         Laptops & computers
+        </a>
```
Comment (denys-danyliuk) [inline src/index.html:37]: 13'th point in checklist

File: src/index.html [hunk 12]
```diff
@@ -6,9 +7,44 @@
...
-    <h1>Moyo header</h1>
+  <!-- #endregion -->
+  <body id="home">
+
+    <header class="header">
+      <a href="#home" class="logo">
+        <img class="logo__img"
+        src="images/logo.png"
+        alt="logo"
+        >
```
Comment (denys-danyliuk) [inline src/index.html:23]: 13'th point in checklist

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]
add test report link, also your demo doesn't look like the design, check it twice
Знімок екрана 2023-07-28 о 11 48 26

File: src/index.html [hunk 13]
```diff
@@ -6,9 +6,61 @@
...
+          <li class="link__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="link__item">
+            <a href="#laptops-and-computers"
+              data-qa="hover"
+              class="nav__link is-active">
+              Laptops & computers
+            </a>
```
Comment (xiimiik) [inline src/index.html:47]: ❗checklist point number '13 ❗

File: src/index.html [hunk 14]
```diff
@@ -6,9 +6,61 @@
...
-  <body>
-    <h1>Moyo header</h1>
+  <body id="home">
+
+    <header class="header">
+      <a href="#home" class="logo">
+        <img class="logo__img"
+        src="images/logo.png"
+        alt="logo"
+        >
```
Comment (xiimiik) [inline src/index.html:21]: ❗checklist point number '13 ❗

File: src/index.html [hunk 15]
```diff
@@ -6,9 +6,61 @@
...
+      </a>
+
+      <nav class="nav">
+        <ul class="list__item">
+          <li class="link__item">
+            <a href="#apple" class="nav__link">Apple</a>
```
Comment (xiimiik) [inline src/index.html:27]: `is-active` should be here

File: src/index.html [hunk 16]
```diff
@@ -6,9 +6,61 @@
...
+          <li class="link__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="link__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+
+          <li class="link__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
```
Comment (xiimiik) [inline src/index.html:39]: `data-qa="hover"` should be here

File: src/index.html [hunk 17]
```diff
@@ -6,9 +6,61 @@
...
+    <header class="header">
+      <a href="#home" class="logo">
+        <img class="logo__img"
+        src="images/logo.png"
+        alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="list__item">
```
Comment (xiimiik) [inline src/index.html:25]:
```suggestion
        <ul class="nav__list">
```

File: src/index.html [hunk 18]
```diff
@@ -6,9 +6,61 @@
...
+          <li class="link__item">
```
Comment (xiimiik) [inline src/index.html:26]:
```suggestion
          <li class="nav__item">
```

File: src/style.css [hunk 19]
```diff
@@ -1,3 +1,60 @@
...
+  border-radius: 3px;
+  width: 100%;
+  height: 4px;
+}
+
+.is-active:hover:after {
+  background-color: #00acdc;
+  height: 4px;
+  margin-bottom: -4px;
+}
```
Comment (xiimiik) [inline src/style.css:48]: delete this

File: src/style.css [hunk 20]
```diff
@@ -1,3 +1,60 @@
...
+  padding: 10px 0;
+  text-decoration: none;
+}
+
+.list__item {
+  display: flex;
+  list-style: none;
+}
+
+.nav__link::after {
```
Comment (xiimiik) [inline src/style.css:34]: these styles must be for the `is-active::after` class

**Review by xiimiik — CHANGES_REQUESTED** [review]
list shouldnt have margin and padding
image
header height is too big
image
edge elements should not have indents along the edges
image
the height of the link should be the full height of the header, so last time I asked you to remove 10px padding from the header
image
the spacing between elements is too large, which is shown even in tests
image

**Comment by ViktoriiaMarkus** [comment]
I dont know what is it but in my laptop everything was correct, 6 tests was passed and 2 failed but here are 8 failed

**Pull request closed without merging**